src/atoms/typography/heading-icon.test.tsx

```diff
@@ -1,5 +1,20 @@
 import React from "react";
+import faker from "faker";
+import { HeadingIcon } from "./heading-icon";
+import { render } from "@testing-library/react";
+import { Icons } from "../constants/icons";
 
 describe("HeadingIcon", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/14", () => {});
+    test("when default props, renders Heading with Icon", () => {
```

Comment:
Does this actually test that an icon was rendered? I'd update the test name or add an additional test to cover that an icon is also rendered.